app/routes.py

```diff
@@ -1,2 +1,64 @@
-from flask import Blueprint
+from app import db
+from app.models.planet import Planet
+from flask import Blueprint, jsonify, abort, make_response, request
 
+planets_bp = Blueprint("planets", __name__, url_prefix="/planets")
+
+
+def validate_model(cls, model_id):
+    try:
+        model_id = int(model_id)
+    except:
+        abort(make_response(jsonify({"message":f"{cls.__name__} {model_id} invalid"}), 400))
+    
+    model = cls.query.get(model_id)
+    if not model:
+        abort(make_response(jsonify({"message":f"{cls.__name__} {model_id} not found"}), 404))
+
+    return model
```

Comment:
Great job updating the validate model method to be generic and applicable to any class!